kratos/mpi/python_scripts/distributed_import_model_part_utility.py

```diff
@@ -31,7 +31,8 @@ def ImportModelPart(self):
                 "skip_timer"                                 : true,
                 "ignore_variables_not_in_solution_step_data" : false,
                 "perform_partitioning"                       : true,
-                "partition_in_memory"                        : false
+                "partition_in_memory"                        : false,
+                "partition_sub_model_parts_list"             : false
```

Comment:
Is this flag required here ? Wouldn't the improved process work when only one modelpart is specified in the list ? That is we can also provide the main modelpart in the list too no ? I am asking because I did not find any check if the provided modelpart is actually a sub-modelpart or not !

Reply:
Also where is the `"submodelpart_list"` defined ? In the same settings ? Then may be it should be added to default settings and validated.

Reply:
Yes, I forgot to add that setting here, you are right. WIth respect to the first point, this only works in principle for submodelparts but it could be adapted so it can use the main model part as input in the list too. My initial idea was that if the "partition_sub_model_parts_list" was false, leave everything as it was before.

Reply:
My suggestion:
call the setting `sub_model_parts_partitioned_separately` => this is IMO very clear what is going on, i.e. if it is empty (default, current behavior) then everything is partitioned together.
Then you can add to this list the submodelarts that you want to partition independently
i.e. in your case `Fluid` and `Solid` => this would create three different partitionings: `Fluid` and `Solid` and the rest that is not contained in those specified SubModelParts